### PR TITLE
Add ignoring lines docs

### LINKED ISSUE #62: Document allowlist
> aha it turns out there are loads of features and they seem to only be documented in the CLI help.  detect-secrets scan --update .secrets.baseline --exclude-files go.sum is the incantation I needed

### PR DESCRIPTION
**Pull request by 0atman**
So when developers are tired of always re-creating the baseline, they can fix it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,20 @@ commits are being made against alphagov without protection so
 hopefully in time we can actually report by commit with another 
 hook.
 
+## Ignoring secrets
+1. In-line `# pragma: allowlist secret` In any language's comment format.
+1. When running detect-secrets, with `detect-secrets scan --update .secrets.baseline --exclude-files go.sum`
+1. Add to your repository's `.pre-commit-config.yaml`, eg:
+```
+$ cat .pre-commit-config.yaml
```
Comment (akinnane) [inline README.md]: I don't think you need to use cat here, you can just show the name and contents of the file. However, this *is* a good candidate for a UUOC award. :joy_cat:

**Review by akinnane — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (0atman) [reply README.md]: TIL about the UUOC award! :laughing:

Comment (0atman) [reply README.md]: it's copied and pasted from the official docs, I'm innocent!

**Comment by 0atman** [comment]
@akinnane tweaked the readme based on your feedback, how does it look now? Thanks!

**Review by akinnane — APPROVED** [review]
Great! Thank you

**Pull request merged**